packages/api-2/src/index.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright 2017-2018 @polkadot/api authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { QueryableStorageFunction, QueryableModuleStorage, QueryableStorage, SubmittableExtrinsics, SubmittableModuleExtrinsics, SubmittableExtrinsicFunction } from './types';
+
+import { Observable } from 'rxjs';
+import WsProvider from '@polkadot/api-provider/ws';
+import defaults from '@polkadot/api-rx/defaults';
+import RpcRx from '@polkadot/api-rx/index';
+import { Extrinsics } from '@polkadot/extrinsics/types';
+import extrinsicsFromMeta from '@polkadot/extrinsics/fromMetadata';
+import { Storage } from '@polkadot/storage/types';
+import storageFromMeta from '@polkadot/storage/fromMetadata';
+import { Method } from '@polkadot/types/index';
+import logger from '@polkadot/util/logger';
+
+import SubmittableExtrinsic from './SubmittableExtrinsic';
+
+const l = logger('api-rx');
+
+export default class ApiRx {
+  // @ts-ignore Indirectly decorated inside the constructor
+  private _extrinsics: SubmittableExtrinsics;
+  // @ts-ignore Indirectly decorated inside the constructor
+  private _state: QueryableStorage;
+  private _rpc: RpcRx;
+
+  constructor (wsProvider: WsProvider = new WsProvider(defaults.WS_URL)) {
+    this._rpc = new RpcRx(wsProvider);
+
+    // !#$%@ But yes, handling promises here
+    this.initMetadata().then().catch();
+  }
+
+  get extrinsics (): SubmittableExtrinsics {
```

Comment:
First off, not as nice/compact as state/rpc, but we do the best we can. Basically -

```
// observable that returns all the status info of the extrinsic
api.extrinsics.balances.transfer(<to-addr>, <value>).sign(<account>).submit()
```

Reply:
Since it is basically a transaction here, could be `api.tx.balances....` (Or something similar, hate too much typing, ideas welcome)

Reply:
On the Rust side, extrinsic is often `xt`. I'd go either `xt` or `extrinsics`, the other 2 proposals here might be confusing.